cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,127 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  let worker;
 
+  beforeEach(() => {
+    cy.visit('');
+    cy.task('generateUser').then((generateUser) => {
+      worker = generateUser;
+    });
+  });
+
+  it('should check pagination and Rows count selection.', () => {
+    cy.createUser(worker);
+    cy.createUser(worker);
+    cy.createUser(worker);
+
+    cy.get('.select-wrap').should('contain', '5 rows');
+    cy.get('.select-wrap').should('contain', '25 rows');
+    cy.get('.select-wrap').should('contain', '5 rows');
+    cy.get('select').select('5');
+
+    cy.get('.-next').click();
+    cy.get('[aria-label="jump to page"]').should('have.value', '2');
+
+    cy.get('.-previous').click();
+    cy.get('[aria-label="jump to page"]').should('have.value', '1');
+
+    cy.get('[aria-label="jump to page"]').clear();
+    cy.get('[aria-label="jump to page"]').type('2{enter}');
+    cy.get('[aria-label="jump to page"]').should('have.value', '2');
+  });
+
+  it('should allow to add a new worker', () => {
+    // by command
+    cy.createUser(worker);
```

Comment:
if your command creates a user using UI, you can use only the command. You do not need to use commands and separate methods that do the same